src/SWP/Bundle/MultiTenancyBundle/Doctrine/PHPCR/OrganizationRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * @author Rafał Muszyński <rafal.muszynski@sourcefabric.org>
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt
+ */
+namespace SWP\Bundle\MultiTenancyBundle\Doctrine\PHPCR;
+
+use SWP\Bundle\StorageBundle\Doctrine\ODM\PHPCR\DocumentRepository;
+use SWP\Component\MultiTenancy\Repository\OrganizationRepositoryInterface;
+
+class OrganizationRepository extends DocumentRepository implements OrganizationRepositoryInterface
+{
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
Might be easier to use SoftDeleteable functionality, for which there is already a filter. Deleted entities can be restored. So there is effectively no difference between disabling an entity and deleting it.